en/docs/administer/managing-users-and-roles/managing-user-stores/configuring-secondary-user-stores.md

```diff
@@ -1,87 +1,97 @@
-    <td><p>Use <code>JDBCUserStoreManager</code> for JDBC user stores. The JDBC user store can be configured for read-only mode or read/write mode using the following property: <code>&lt;Property name=&quot;ReadOnly&quot;&gt;false/true&lt;/Property&gt;</code> .</p></td>
+    <td><p>Use <code>JDBCUserStoreManager</code> for JDBC user stores. The JDBC user store can be configured for read-only mode or read/write mode using the following property: <code>&lt;Property name=&quot;ReadOnly&quot;&gt;false/true&lt;/Property&gt;</code>.</p></td>
     </tr>
     </tbody>
     </table>
 
-    You can also populate this drop-down list with custom user store manager implementations by adding them to the server. The next [topic](../writing-a-custom-user-store-manager) includes information on implementing a custom user store manager.
+    You can also populate this drop-down list with custom user store manager implementations by adding them to the server. For information on implementing a custom user store manager, see [Writing a Custom User Store Manager]({{base_path}}/administer/managing-users-and-roles/managing-user-stores/writing-a-custom-user-store-manager).
     
-    We are maintaining samples [here](https://github.com/wso2/product-is/tree/v5.8.0/modules/samples/user-mgt/sample-custome-user-store-manager), under the WSO2 Identity Server product repository, and you can filter that based on the Identity Server version.
+    For samples, see [Sample Custom User Store Manager](https://github.com/wso2/product-is/tree/v5.10.0/modules/samples/user-mgt/sample-custom-user-store-manager) in the WSO2 Identity Server product repository.
+
+4. Enter a unique domain name.
 
-4.  Enter a unique domain name with no underscore (\_) characters, and optionally enter a description for this user store.
+     Do not enter underscore (\_) characters in the domain name, and optionally enter a description for this user store.
 
-5.  Enter values for the properties, using the descriptions in the Descriptions column for guidance. The properties that appear vary based on the user store manager class you selected, and there may be additional properties in Optional or Advanced sections at the bottom of the screen. For information on the properties that are used when defining user stores, see [Properties of User Stores](https://docs.wso2.com/display/ADMIN44x/Working+with+Properties+of+User+Stores) .
+5. Enter values for the properties, using the descriptions in the Descriptions column for guidance. 
 
-    ![]({{base_path}}/assets/img/administer/secondary-user-store-definition.png)
+     The properties vary based on the user store manager class you selected, and there may be additional properties in Optional or Advanced sections at the bottom of the screen. For information on the properties that are used when defining user stores, see [Properties of User Stores](https://docs.wso2.com/display/ADMIN44x/Working+with+Properties+of+User+Stores).
```

Comment:
This should not point to the Admin guide. Should we move this entire page to the APIM documentation?